docs/getting-started/overview.rst

```diff
@@ -1,6 +1,113 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. This document will outline the current state of the .NET ecosystem and its various 
+implementations, so you can get a good "lay of the land". 
+
+.NET Core
+---------
+.NET Core is a cloud-optimized, cross-platform implementation of the .NET Platform. It currently 
+supports three main operating systems: Linux, Windows and OS X. 
+
+There are several reasons why we built .NET Core.
+
+**Cross-platform support** is a requirement of many developers today. It's important to use the 
```

Comment:
I'd say a word about dev boxes and production environments, and how depite being commonly running a different OS, applications still need to run reliably unchanged between them.